configure.ac

```diff
@@ -36,5 +36,10 @@ AC_FUNC_MALLOC
 AC_FUNC_REALLOC
 AC_CHECK_FUNCS([bzero gethostbyname inet_ntoa memmove memset memmem select socket strerror strstr strtol])
 
+AC_DEFINE_UNQUOTED([PREFIX],["$prefix"])
+AC_DEFINE_UNQUOTED([SYSCONFDIR],["`echo $(eval echo $sysconfdir)`"])
```

Comment:
for `configure --sysconfdir` it would be enough, but for `configure --prefix` a simple `echo $sysconfdir` would give `${prefix}/etc` for `@SYSCONFDIR@`. The `eval` is needed here, but I could simplify this to `` `eval echo $sysconfdir` ``. I have chosen the backtick notation for better portability, because we need just one level of execution now (even though the `$( ... )` notation should be widely supported nowadays).